middleware/applyOverrides.js

```diff
@@ -0,0 +1,41 @@
+const _ = require('lodash');
+const logger = require('pelias-logger').get('api');
+const codec = require('pelias-model').codec;
+
+function setup() {
```

Comment:
can we remove this function and `module.exports = applyOverrides` instead?

Reply:
we could but that doesn't fit with the style of all the other middleware that's called as a function when being added to the call chain in v1.js

ex:
      middleware.applyOverrides(),
      middleware.dedupe(),
      middleware.accuracy(),
      middleware.localNamingConventions(),
      middleware.renamePlacenames(),
      middleware.parseBBox(),
      middleware.normalizeParentIds(),

Reply:
Agh ok, fair enough.
Seems like the right thing to do to have an outer function which can optionally accept args, kinda like a constructor.
In this case it's not required but you're right, it makes them more consistent.